TradeItIosTicketSDK2/TradeItSymbolSearchViewController.swift

```diff
@@ -15,7 +15,6 @@ class TradeItSymbolSearchViewController: TradeItViewController, UITableViewDeleg
 
         self.activityIndicator.hidesWhenStopped = true
         setupSearchTextField()
-
```

Comment:
There should be vertical whitespace separating both explicit blocks of code and also implicit bocks of code to aid in visually organizing business logic.